packages/sanity/src/_internal/cli/actions/schema/schemaListAction.ts

```diff
@@ -0,0 +1,100 @@
+import {type CliCommandArguments, type CliCommandContext, type CliOutputter} from '@sanity/cli'
+import {type SanityDocument} from '@sanity/client'
+import chalk from 'chalk'
+import {size, sortBy} from 'lodash'
+
+export interface SchemaListFlags {
+  json: boolean
+  id: string
+}
+
+type PrintSchemaListArgs = {
+  schemas: SanityDocument[]
+  output: CliOutputter
+  dataset: string
+  projectId: string
+}
+
+export const SANITY_WORKSPACE_SCHEMA_ID = 'sanity.workspace.schema'
+
+const printSchemaList = ({schemas, output, dataset, projectId}: PrintSchemaListArgs) => {
+  const ordered = sortBy(
+    schemas.map(({_createdAt: createdAt, _id: id, workspace}) => {
+      return [id, workspace.title, dataset, projectId, createdAt].map(String)
```

Comment:
I still maintain that it should be "Workspace name" and not title. workspace.title is not validated to be unique in sanity.config, so you could have duplicates here.